src/components/ArticleInfo.jsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { fetchArticleById } from "../../api";
+import { Link } from "react-router-dom";
+
+export default function ArticleInfo() {
+  const { articleId } = useParams();
+  const [loading, setLoading] = useState(true);
+  const [article, setArticle] = useState({});
+
+  useEffect(() => {
+    setLoading(true);
+    fetchArticleById(articleId).then((article) => {
+      setArticle(article);
+      setLoading(false);
+    });
+  }, [articleId]);
+
+  const articleDate = new Date(article.created_at);
+
+  const formattedDate = articleDate.toLocaleString();
+
+  return loading ? (
+    <h2>Loading...</h2>
+  ) : (
+    <section>
+      <div className="article-information">
```

Comment:
this `div` is unnecessary. Can't you put the `className` in the `section` above?